applications/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ if (BUILD_APPLICATIONS)
     target_link_libraries(grdecl_grid   ecl)
     target_link_libraries(summary       ecl)
 
+    if (BUILD_NEXUS)
+        add_executable(nex2ecl   nexus/nex2ecl.cpp)
+        target_link_libraries(nex2ecl   ecl)
+    endif ()
```

Comment:
Need to add the `nex2ecl` binary to the `apps` list to ensure it is installed:
```txt
list( APPEND apss nex2ecl )
```

Reply:
OK. Fixed now. But the executable was already in `build/bin` and working?

Reply:
Yes - but when you later did `make install` it would not have installed in `$PREFIX/bin`